### PR TITLE
Boosts Headline padding to 40px (32px on mobile)

### LINKED ISSUE #375: Headline should have 40px padding
**Is your feature request related to a problem? Please describe.**
As per designs, headline should have 40px padding top and bottom (tested on iPad mini)"

Looks like mobile should have 32px padding.

**Describe the solution you'd like**
A PR which changes the `@bbc/psammead-headings` component to accommodate the new designs.
Designs can be seen [here](https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5c596fa94e189735593e60e8) 

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: check in Storybook, should see extra padding. Snapshots should be updated.

**Additional context**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Resolves #375

**Overall change:** Changes Headline padding according to the designs.

**Code changes:**

- Increases padding to 40px for Smartphone and up. 32px for smaller.
- Updates snapshot tests
- Adds changelog and bumps version number

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by twinlensreflex — DISMISSED** [review]

File: packages/components/psammead-headings/src/index.jsx [hunk 1]
```diff
@@ -10,12 +10,16 @@ import {
...
 } from '@bbc/gel-foundations/typography';
+import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-foundations/breakpoints';
 
 export const Headline = styled.h1`
   color: ${C_SHADOW};
   font-family: ${GEL_FF_REITH_SERIF};
   margin: 0; /* Reset */
-  padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_DBL} 0;
+  padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_QUAD} 0;
+  ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_AND_LARGER} {
```
Comment (dr3) [inline packages/components/psammead-headings/src/index.jsx]: The Zeplin has it use 32px at 375px, however this doesnt do it until 318px, likely need to use the next breakpoint up

**Review by dr3 — CHANGES_REQUESTED** [review]

File: packages/components/psammead-headings/src/index.jsx (hunk 1 above)
Comment (ChrisBAshton) [reply packages/components/psammead-headings/src/index.jsx]: Hmmm `LAPTOP_AND_LARGER` is `min-width: 600px`, so is the right one to use in this case - will update 👍 .

But the name is confusing - 600px wide is smaller than your typical _tablet_, so `LAPTOP_AND_LARGER` makes no sense 🤔

**Review by dr3 — CHANGES_REQUESTED** [review]
LGTM, just needs a package bump and changelog

File: packages/components/psammead-headings/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Psammead Headings Changelog
 
+<!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 0.3.2   | [PR#381](https://github.com/bbc/psammead/pull/381) Make headline 44px for 600px and above |
```
Comment (dr3) [inline packages/components/psammead-headings/CHANGELOG.md]:
```suggestion
| 0.3.2   | [PR#381](https://github.com/bbc/psammead/pull/381) Make headline 40px for 600px and above |
```

File: packages/components/psammead-headings/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-headings",
-  "version": "0.3.1",
+  "version": "0.3.2",
```
Comment (dr3) [inline packages/components/psammead-headings/package.json]: Visual change should be `0.4.0` IMO

File: packages/components/psammead-headings/src/index.jsx [hunk 4]
```diff
@@ -10,12 +10,16 @@ import {
...
   GEL_FF_REITH_SERIF,
 } from '@bbc/gel-foundations/typography';
+import { MEDIA_QUERY_TYPOGRAPHY } from '@bbc/gel-foundations/breakpoints';
 
 export const Headline = styled.h1`
   color: ${C_SHADOW};
   font-family: ${GEL_FF_REITH_SERIF};
   margin: 0; /* Reset */
-  padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_DBL} 0;
+  padding: ${GEL_SPACING_QUAD} 0 ${GEL_SPACING_QUAD} 0;
```
Comment (dr3) [inline packages/components/psammead-headings/src/index.jsx]:
```suggestion
  padding: ${GEL_SPACING_QUAD} 0;
```

**Review by dr3 — DISMISSED** [review]

**Review by pjlee11 — DISMISSED** [review]

**Comment by jamesbrumpton** [comment]
👍

**Review by pjlee11 — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**